Resources/Prototypes/GameRules/cargo_gifts.yml

```diff
@@ -1,6 +1,15 @@
 - type: entity
-  id: GiftsPizzaPartySmall
+  id: CargoGiftsBase
   parent: BaseGameRule
+  abstract: true
+  components:
+  - type: StationEvent
+    startAudio:
+      path: /Audio/Announcements/announce.ogg
```

Comment:
im pretty sure you can put more repeated info shared between all children in here like CargoGiftsRule.Sender and maybe other stuff

Reply:
Okay